packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -187,6 +188,21 @@ export default function NavigationPlaceholder( {
 
 	const { navigationMenus } = useNavigationMenu();
 
+	const dividerProps = {
+		borderColor: 'black',
+		orientation: 'vertical',
+	};
+	const dividerStyles = {
```

Comment:
If I set the orientation to vertical, it seems like the component could be smart enough that the only styles I needed to define were _height_ (if I wanted anything other than `auto`), and possibly left/right margins. It seems like we could provide nice defaults for margins and colors so those would be accurate without overriding in most cases.